lib/Twig/Extensions/Extension/Gettext/Token.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+class Twig_Extensions_Extension_Gettext_Token extends Twig_Token {
+    
+    const COMMENT = 12;
+    
+    public static function typeToString($type, $short = false, $line = -1) {
+        switch ($type) {
+            case self::COMMENT :
+                $name = 'COMMENT';
+                break;
+            default :
+                return parent::typeToString($type, $short, $line);
+        }
+
+        return $short ? $name : 'Twig_Token::'.$name;
```

Comment:
It is defined above, but as ``Twig_Extensions_Extension_Gettext_Token::COMMENT``, not as ``Twig_Token::COMMENT``